biocypher_metta/adapters/dbsuper_adapter.py

```diff
@@ -0,0 +1,90 @@
+import csv
+import gzip
+import pickle
+from biocypher_metta.adapters import Adapter
+from liftover import get_lifter
+
+from biocypher_metta.adapters.helpers import check_genomic_location
+# Example dbSuper tsv input files:
```

Comment:
what does the `rank` column refer to?

Reply:
The rank assigned to the super enhancer within that specific cell type based on the [H3K27ac ChIP-seq](https://en.wikipedia.org/wiki/H3K27ac) signal strength. Higher ranks indicate a stronger signal and potentially a more potent super enhancer.